kolibri/core/templates/kolibri/base.html

```diff
@@ -18,10 +15,11 @@
 </ul>
 {% endblock %}
 
-<hr>
-
-{% block content %}
-
+<main></main>
+{% block frontend_assets %}
```

Comment:
The ready method of the learn KolibriModule must be executed *after* the `<main>` tag is created... which is not the case if that code is included in the `<head>`